packages/@aws-cdk/aws-appmesh/README.md

```diff
@@ -307,3 +307,21 @@ gateway.addGatewayRoute('gateway-route-grpc', {
   }),
 });
 ```
+
+## Importing Resources
+
+Each resource comes with two static methods for importing a reference to an existing App Mesh resource.
+There are two static methods, `from<Resource>Arn` and `from<Resource>Attributes` where the `<Resource>` is replaced with the resource name.
```

Comment:
I guess `Mesh` is an exception to the rule...?

Reply:
hm. Fair point..

Reply:
Updated to include `Mesh`